schema/objects/Financing.schema.json

```diff
@@ -0,0 +1,34 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "https://raw.githubusercontent.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/main/schema/objects/Financing.schema.json",
+  "title": "Object - Financing",
+  "description": "Object describing a financing",
+  "type": "object",
+  "allOf": [
+    {
+      "$ref": "https://raw.githubusercontent.com/Open-Cap-Table-Coalition/Open-Cap-Format-OCF/main/schema/primitives/objects/Object.schema.json"
+    }
+  ],
+  "properties": {
+    "object_type": {
+      "const": "FINANCING"
+    },
+    "name": {
+      "description": "Name for the financing",
+      "type": "string"
+    },
+    "id": {},
+    "issuance_ids": {
+      "description": "Array of issuance IDs associated with the financing",
+      "type": "array",
+      "items": {
+        "type": "string",
+        "minItems": 1
+      }
+    },
+    "comments": {}
+  },
+  "additionalProperties": false,
+  "required": ["name", "issuance_ids"],
```

Comment:
Should there be a `date` associated with each financing?

Reply:
The sample file mentions the date in the comments section
```
{
  "file_type": "OCF_FINANCINGS_FILE",
  "items": [
    {
      "object_type": "FINANCING",
      "id": "5bfe12c6-f049-4216-9f94-5a75dc3aaee5",
      "name": "Series A Funding",
      "issuance_ids": ["test-stock-issuance-minimal"],
      "comments": ["A funding round led by Firm Y on 02/15/2024."]
    }
  ]
}
```
In my opinion this data should be structured